manage/upgrading/version_specific_migration/upgrade_zodb_to_python3.rst

```diff
@@ -33,7 +33,7 @@ In short you need to follow these steps to migrate your database:
 #. Make sure your code and all add-ons that you use work in Python 3.
    (see :doc:`upgrade_to_python3`)
 #. Backup your database!
-#. Pack your database.
+#. **Pack** your database to **0 days** (`zodbupdate` won't update your database history and will leave old objects in place and you won't be able to pack your database in the future).
```

Comment:
reStructuredText uses double backticks around inline literals. markdown uses single.

```
``zodbupdate``
```

Also avoid contractions:

s/won't/will not.

Reply:
@stevepiercy I just found the GitHub "suggestion" feature and thought it could simplify your suggestions, since you often suggest changes - as I do too. See bullet number 6 at this link: https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
oooo, neato! Thanks for the tip, @jensens. That will save me a lot of manual typing.